catalyst/README.md

```diff
@@ -153,158 +158,413 @@ whenItem(store.buttonParent)
 
 rule.subscribe(store.buttonParent)
 
-
 Or to specific function that span variants
 
-```
+```js
 rule.app.createMainButton = function(){
-rule
-.whenDOM('.evolv-buttonParent')
-.then(function(buttonParent){
-	buttonParent.append(...)
+  rule
+    .whenDOM('.evolv-buttonParent')
+    .then(function(buttonParent){
+      buttonParent.append(...);
+    }
 };
 ```
 
 ## Modes
```

Comment:
I don't think you need a Modes heading, I'd just delete it, it's messing up your heading heirarchy